a3c/checkpoints/latest_test/config_list.txt
Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, mean=Close_x           6.957138e+02
Averageclose_x    6.940133e+02
Volume_x          1.277590e+08
Close_y           1.418135e+01
Averageclose_y    1.398464e+01
Volume_y          3.167284e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x           2.613592e+02
Averageclose_x    2.573756e+02
Volume_x          1.117408e+08
Close_y           1.257827e+01
Averageclose_y    1.189683e+01
Volume_y          5.424033e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, mean=Close_x           6.957138e+02
Averageclose_x    6.940133e+02
Volume_x          1.277590e+08
Close_y           1.418135e+01
Averageclose_y    1.398464e+01
Volume_y          3.167284e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x           2.613592e+02
Averageclose_x    2.573756e+02
Volume_x          1.117408e+08
Close_y           1.257827e+01
Averageclose_y    1.189683e+01
Volume_y          5.424033e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, mean=Close_x           6.957138e+02
Averageclose_x    6.940133e+02
Volume_x          1.277590e+08
Close_y           1.418135e+01
Averageclose_y    1.398464e+01
Volume_y          3.167284e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x           2.613592e+02
Averageclose_x    2.573756e+02
Volume_x          1.117408e+08
Close_y           1.257827e+01
Averageclose_y    1.189683e+01
Volume_y          5.424033e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x           6.957138e+02
Averageclose_x    6.940133e+02
Volume_x          1.277590e+08
Close_y           1.418135e+01
Averageclose_y    1.398464e+01
Volume_y          3.167284e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x           2.613592e+02
Averageclose_x    2.573756e+02
Volume_x          1.117408e+08
Close_y           1.257827e+01
Averageclose_y    1.189683e+01
Volume_y          5.424033e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x           6.957138e+02
Averageclose_x    6.940133e+02
Volume_x          1.277590e+08
Close_y           1.418135e+01
Averageclose_y    1.398464e+01
Volume_y          3.167284e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x           2.613592e+02
Averageclose_x    2.573756e+02
Volume_x          1.117408e+08
Close_y           1.257827e+01
Averageclose_y    1.189683e+01
Volume_y          5.424033e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x           6.957138e+02
Averageclose_x    6.940133e+02
Volume_x          1.277590e+08
Close_y           1.418135e+01
Averageclose_y    1.398464e+01
Volume_y          3.167284e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x           2.613592e+02
Averageclose_x    2.573756e+02
Volume_x          1.117408e+08
Close_y           1.257827e+01
Averageclose_y    1.189683e+01
Volume_y          5.424033e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=True, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x           6.957138e+02
Averageclose_x    6.940133e+02
Volume_x          1.277590e+08
Close_y           1.418135e+01
Averageclose_y    1.398464e+01
Volume_y          3.167284e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x           2.613592e+02
Averageclose_x    2.573756e+02
Volume_x          1.117408e+08
Close_y           1.257827e+01
Averageclose_y    1.189683e+01
Volume_y          5.424033e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=2, input_size=4, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x          695.713799
priceratio_x       0.004928
volumeratio_x      0.018625
Close_y           14.181350
priceratio_y       0.019177
volumeratio_y      0.042467
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x          261.359206
priceratio_x       0.030778
volumeratio_x      0.320233
Close_y           12.578270
priceratio_y       0.080898
volumeratio_y      0.489649
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=2, input_size=4, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x          695.713799
priceratio_x       0.004928
volumeratio_x      0.018625
Close_y           14.181350
priceratio_y       0.019177
volumeratio_y      0.042467
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x          261.359206
priceratio_x       0.030778
volumeratio_x      0.320233
Close_y           12.578270
priceratio_y       0.080898
volumeratio_y      0.489649
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=2, input_size=4, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x          695.713799
priceratio_x       0.004928
volumeratio_x      0.018625
Close_y           14.181350
priceratio_y       0.019177
volumeratio_y      0.042467
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x          261.359206
priceratio_x       0.030778
volumeratio_x      0.320233
Close_y           12.578270
priceratio_y       0.080898
volumeratio_y      0.489649
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x          695.713799
priceratio_x       0.004928
volumeratio_x      0.018625
Close_y           14.181350
priceratio_y       0.019177
volumeratio_y      0.042467
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x          261.359206
priceratio_x       0.030778
volumeratio_x      0.320233
Close_y           12.578270
priceratio_y       0.080898
volumeratio_y      0.489649
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=2, input_size=4, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x          695.713799
priceratio_x       0.004928
volumeratio_x      0.018625
Close_y           14.181350
priceratio_y       0.019177
volumeratio_y      0.042467
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x          261.359206
priceratio_x       0.030778
volumeratio_x      0.320233
Close_y           12.578270
priceratio_y       0.080898
volumeratio_y      0.489649
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=2, input_size=4, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x          695.713799
priceratio_x       0.004928
volumeratio_x      0.018625
Close_y           14.181350
priceratio_y       0.019177
volumeratio_y      0.042467
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x          261.359206
priceratio_x       0.030778
volumeratio_x      0.320233
Close_y           12.578270
priceratio_y       0.080898
volumeratio_y      0.489649
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x          695.713799
priceratio_x       0.004928
volumeratio_x      0.018625
Close_y           14.181350
priceratio_y       0.019177
volumeratio_y      0.042467
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x          261.359206
priceratio_x       0.030778
volumeratio_x      0.320233
Close_y           12.578270
priceratio_y       0.080898
volumeratio_y      0.489649
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x            6.957138e+02
Averageclose_x     6.940133e+02
Averagevolume_x    1.264072e+08
Close_y            1.418135e+01
Averageclose_y     1.398464e+01
Averagevolume_y    3.074367e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x            2.613592e+02
Averageclose_x     2.573756e+02
Averagevolume_x    9.709476e+07
Close_y            1.257827e+01
Averageclose_y     1.189683e+01
Averagevolume_y    4.502809e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=400000, mean=Close_x            6.957138e+02
Averageclose_x     6.940133e+02
Averagevolume_x    1.264072e+08
Close_y            1.418135e+01
Averageclose_y     1.398464e+01
Averagevolume_y    3.074367e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x            2.613592e+02
Averageclose_x     2.573756e+02
Averagevolume_x    9.709476e+07
Close_y            1.257827e+01
Averageclose_y     1.189683e+01
Averagevolume_y    4.502809e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=False, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, mean=Close_x            6.957138e+02
Averageclose_x     6.940133e+02
Averagevolume_x    1.264072e+08
Close_y            1.418135e+01
Averageclose_y     1.398464e+01
Averagevolume_y    3.074367e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x            2.613592e+02
Averageclose_x     2.573756e+02
Averagevolume_x    9.709476e+07
Close_y            1.257827e+01
Averageclose_y     1.189683e+01
Averagevolume_y    4.502809e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=True, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, mean=Close_x            6.957138e+02
Averageclose_x     6.940133e+02
Averagevolume_x    1.264072e+08
Close_y            1.418135e+01
Averageclose_y     1.398464e+01
Averagevolume_y    3.074367e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x            2.613592e+02
Averageclose_x     2.573756e+02
Averagevolume_x    9.709476e+07
Close_y            1.257827e+01
Averageclose_y     1.189683e+01
Averagevolume_y    4.502809e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=3, alloRNN_fc0_output_dim=64, asset_num=2, checkpoint_dir='./checkpoints', dropout=True, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.],
       [ 0.,  1.]]), grad_norm_clip=40.0, info_num=3, input_size=6, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, mean=Close_x            6.957138e+02
Averageclose_x     6.940133e+02
Averagevolume_x    1.264072e+08
Close_y            1.418135e+01
Averageclose_y     1.398464e+01
Averagevolume_y    3.074367e+07
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=Close_x            2.613592e+02
Averageclose_x     2.573756e+02
Averagevolume_x    9.709476e+07
Close_y            1.257827e+01
Averageclose_y     1.189683e+01
Averagevolume_y    4.502809e+07
dtype: float64, test_name='latest_test', test_steps=5000, thread_num=4, use_checkpoint=False)Namespace(action_size=12, alloRNN_fc0_output_dim=64, asset_num=11, checkpoint_dir='./checkpoints', dropout=True, entropy_beta=0.01, game='CartPole-v0', gamma=0.9999, gauss_sigma=array([[ 1.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  1.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  1.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  1.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  1.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  1.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  1.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  1.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  1.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  1.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  1.]]), grad_norm_clip=40.0, info_num=4, input_size=44, keep_prob=0.5, learning_rate=0.0005, local_t_max=32, lstm1_unit=15, lstm_unit=64, max_time_step=1000000, mean=CLOSE_x              1.592613e+02
VOLUME_x             4.041820e+09
inputClose_x         1.592613e+02
Averageclose_x       1.592588e+02
Averagevolume_x      4.010684e+09
CLOSE_y              2.995942e+03
VOLUME_y             1.475143e+10
inputClose_y         2.995942e+03
Averageclose_y       2.996269e+03
Averagevolume_y      1.478055e+10
CLOSE_x.1            5.418295e+03
VOLUME_x.1           8.123628e+09
inputClose_x.1       5.418295e+03
Averageclose_x.1     5.420950e+03
Averagevolume_x.1    8.137854e+09
CLOSE_y.1            3.417279e+03
VOLUME_y.1           2.287505e+10
inputClose_y.1       3.417279e+03
Averageclose_y.1     3.418022e+03
Averagevolume_y.1    2.291840e+10
CLOSE_x.2            2.032782e+03
VOLUME_x.2           5.137011e+09
inputClose_x.2       2.032782e+03
Averageclose_x.2     2.032606e+03
Averagevolume_x.2    5.146610e+09
CLOSE_y.2            1.795175e+03
VOLUME_y.2           2.713256e+09
inputClose_y.2       1.795175e+03
Averageclose_y.2     1.797061e+03
Averagevolume_y.2    2.717173e+09
CLOSE_x.3            8.226195e+02
VOLUME_x.3           7.417297e+04
inputClose_x.3       8.226195e+02
Averageclose_x.3     8.223000e+02
Averagevolume_x.3    7.424222e+04
CLOSE_y.3            1.841843e+03
VOLUME_y.3           7.136474e+04
inputClose_y.3       1.841843e+03
Averageclose_y.3     1.840014e+03
Averagevolume_y.3    7.144321e+04
CLOSE_x.4            1.044686e+03
VOLUME_x.4           5.384818e+04
inputClose_x.4       1.044686e+03
Averageclose_x.4     1.043141e+03
Averagevolume_x.4    5.386559e+04
CLOSE_y.4            6.091671e+02
VOLUME_y.4           1.445515e+04
inputClose_y.4       6.091671e+02
Averageclose_y.4     6.086123e+02
Averagevolume_y.4    1.449239e+04
CLOSE                1.159171e+03
VOLUME               2.142942e+03
inputClose           1.159171e+03
Averageclose         1.158406e+03
Averagevolume        2.146276e+03
dtype: float64, only_train_positive=True, risk_beta=0.1, rmsp_alpha=0.99, rmsp_epsilon=0.1, share_variable=True, short_sight=False, short_sight_step=30, sigma=1.0, state_feature_num=64, std=CLOSE_x              1.209951e+01
VOLUME_x             4.855249e+09
inputClose_x         1.209951e+01
Averageclose_x       1.208250e+01
Averagevolume_x      3.944526e+09
CLOSE_y              7.207734e+02
VOLUME_y             1.226651e+10
inputClose_y         7.207734e+02
Averageclose_y       7.197593e+02
Averagevolume_y      1.190004e+10
CLOSE_x.1            1.713365e+03
VOLUME_x.1           4.418007e+09
inputClose_x.1       1.713365e+03
Averageclose_x.1     1.709044e+03
Averagevolume_x.1    4.228695e+09
CLOSE_y.1            8.859194e+02
VOLUME_y.1           1.629433e+10
inputClose_y.1       8.859194e+02
Averageclose_y.1     8.845033e+02
Averagevolume_y.1    1.580504e+10
CLOSE_x.2            4.621519e+02
VOLUME_x.2           5.187859e+09
inputClose_x.2       4.621519e+02
Averageclose_x.2     4.614554e+02
Averagevolume_x.2    4.996900e+09
CLOSE_y.2            6.435249e+02
VOLUME_y.2           1.539196e+09
inputClose_y.2       6.435249e+02
Averageclose_y.2     6.411627e+02
Averagevolume_y.2    1.487304e+09

a3c/config.py
import argparse
import numpy as np
import os

cwd = os.getcwd()
parser = argparse.ArgumentParser()

# env parameters
parser.add_argument('--game', type=str, default='CartPole-v0',
                    help='Name of the atari game to play. Full list here: https://gym.openai.com/envs#atari')

# input parameters
parser.add_argument('--asset_num', type=int, default=11)
parser.add_argument('--info_num', type=int, default=3)
parser.add_argument('--input_size', type=int, default=33,
                    help='input size = asset num * info num')

# model parameters
parser.add_argument('--share_variable', type=bool, default=True,
                    help='wether use the independent lstm')
parser.add_argument('--dropout', type=bool, default=True,
                    help='use droup_out')
parser.add_argument('--short_sight', type=bool, default=False,
                    help='lookfoward only a few steps')
parser.add_argument('--lstm1_unit', type=int, default=256,
                    help='the output size of indepent_lstm1')
parser.add_argument('--lstm_unit', type=int, default=128,
                    help='the output size of lstm')
parser.add_argument('--state_feature_num', type=int, default=128,
                    help='the num of feature extracted from both state and allocation')
parser.add_argument('--alloRNN_fc0_output_dim', type=int, default=128,
                    help='the dimention of fc0 of the direct allocation RNN')
parser.add_argument('--keep_prob', type=float, default=0.5,
                    help='keep probability in droup out')
parser.add_argument('--short_sight_step', type=int, default=30,
                    help='lookfoward steps')
parser.add_argument('--entropy_beta', type=float, default=0.01)

# finance parameters
parser.add_argument('--gamma', type=float, default=0.9999,
                    help='daily discount rate, 0.9999 equals to capital return rate 103.7% per year')
parser.add_argument('--risk_beta', type=float, default=0.1,
                    help='the multiplier for gauss mean l1 loss, represents the risk preference, greater risk_beta means safer')
# train parameters
parser.add_argument('--local_t_max', type=int, default=32,
                    help='async interval of a single thread. In fact it is the same as batch size')
parser.add_argument('--max_time_step', type=int, default=10 * 10 ** 5)
parser.add_argument('--learning_rate', type=float, default=0.0005)
parser.add_argument('--thread_num', type=int, default=4)
parser.add_argument('--sigma', type=float, default=1.0,
                    help='init sigma as sigme*I')

# log parameters
# the directory saved ckpt is joined by two parts
# the parent dir "checkpoint" and the identical dir "test_name"
# this design is for the convenient of various test
parser.add_argument('--checkpoint_dir', type=str, default='./checkpoints')
parser.add_argument('--use_checkpoint', type=bool, default='False')
parser.add_argument('--test_steps', type=int, default=5000)
parser.add_argument('--test_name', type=str, default='latest_test')

# gradient applier parameters
parser.add_argument('--rmsp_alpha', type=float, default=0.99)
parser.add_argument('--rmsp_epsilon', type=float, default=0.1)
parser.add_argument('--grad_norm_clip', type=float, default=40.0)

args = parser.parse_args()

# additional parameters
args.action_size = args.asset_num + 1
args.use_checkpoint = False
args.gauss_sigma = args.sigma * np.eye(args.action_size - 1)
args.only_train_positive = True

args.mean = 0
args.std = 0
